apps/frontpage/components/home/social-validation/social-validation.tsx

```diff
@@ -334,7 +336,7 @@ export const SocialValidation: FC = () => {
             <a href="https://twitter.com/storybookjs">Follow on Twitter</a>
           </Button>
           <div>
-            <div className="text-sm text-white">{`${twitterFollowerCount.toLocaleString()}+`}</div>
+            <div className="text-sm text-white">24,200+</div>
```

Comment:
I'd rather keep these hardcoded values as constants, to aid semi-regular updates.

```suggestion
            <div className="text-sm text-white">{`${TWITTER_FOLLOWER_COUNT}+`}</div>
```

Reply:
Alternatively, you could go ahead and implement `getTwitterFollowerCount` and just have it return a constant. Either approach is fine, imo.

Reply:
I don't think this make sense to make them a constant if this is to be used only in one place. You make them a constant if you want to use them in multiple places.